src/main/java/com/gotocompany/depot/http/request/parser/JsonElementParser.java

```diff
@@ -0,0 +1,31 @@
+package com.gotocompany.depot.http.request.parser;
+
+import com.google.gson.JsonElement;
+import com.gotocompany.depot.message.ParsedMessage;
+
+public interface JsonElementParser {
+
+    String parse(JsonElement jsonElement, ParsedMessage parsedMessage);
+
+
+    static JsonElementParser getParser(JsonElement jsonElement) {
```

Comment:
this is weird pattern. Let's put this method outside of this interface.

Reply:
fixed